breathecode/mentorship/views.py

```
@@ -689,20 +690,35 @@ def get(self, request, bill_id=None, academy_id=None):
 
     @capable_of('read_mentorship_bill')
     def put(self, request, bill_id=None, academy_id=None):
+        many = isinstance(request.data, list)
+        if many:
+            current = []
+            for obj in request.data:
+                current.append(MentorshipBill.objects.filter(id=obj['id']).first())
```

If a element not found, it should raise a exception